python-package/xgboost/sklearn.py

```diff
@@ -1497,6 +1553,15 @@ def _more_tags(self) -> Dict[str, bool]:
         tags["multilabel"] = True
         return tags
 
+    def __sklearn_tags__(self) -> _sklearn_Tags:
+        tags = XGBModel.__sklearn_tags__(self)
+        tags.estimator_type = "classifier"
```

Comment:
Do we need this if we inherit the classifier mixin?

Reply:
Ah you are totally right, I don't think we do:

https://github.com/scikit-learn/scikit-learn/blob/e6037ba412ed889a888a60bd6c022990f2669507/sklearn/base.py#L536-L544

Removed this, the corresponding `LGBRegressor` code, and the imports of `ClassifierTags` / `RegressorTags` in https://github.com/dmlc/xgboost/pull/11021/commits/a51184888a7cf1e80c9b262a5af2f2518ca53ebc